src/components/About.tsx

```diff
@@ -0,0 +1,41 @@
+type AboutProps = {
+    photoUrl?: string; // optional for now
+};
+
+export default function About({ photoUrl }: AboutProps) {
+    return (
+        <section id="about" className="bg-[#f4f3ef] py-16 px-6 md:px-20 text-gray-800">
```

Comment:
- [ ] Colours still need to be added to the tailwind theme instead of hardcoded like this